src/translation/translation_nb_NO.ts

```diff
@@ -210,36 +210,36 @@
         <location filename="../audiomixerboard.cpp" line="236"/>
         <location filename="../audiomixerboard.cpp" line="250"/>
         <source>Pan</source>
-        <translation type="unfinished"></translation>
+        <translation>Panorering</translation>
```

Comment:
I'd check how this looks in the app @comradekingu since Panorering might be too big.

Reply:
Yes, the checkbox and label want to be as small as possible.

Reply:
There is a https://docs.weblate.org/en/latest/user/checks.html?highlight=max#maximum-length-of-translation
available.
I think it is also possible to take into account how that factors into the given font used, but a sensible limit helps sort this for now.